cmd/gctcli/commands.go

```diff
@@ -2049,3 +2050,99 @@ func setLoggerDetails(c *cli.Context) error {
 	jsonOutput(result)
 	return nil
 }
+
+var getOrderbookStreamCommand = cli.Command{
+	Name:      "getorderbookstream",
+	Usage:     "gets the orderbook stream for a specific currency pair and exchange",
+	ArgsUsage: "<exchange> <base> <quote> <asset>",
+	Action:    getOrderbookStream,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "exchange",
+			Usage: "the exchange to get the ticker for",
+		},
+		cli.StringFlag{
+			Name:  "base",
+			Usage: "base currency",
+		},
+		cli.StringFlag{
+			Name:  "quote",
+			Usage: "quote currency",
+		},
+		cli.StringFlag{
+			Name:  "asset",
+			Usage: "the asset type of the currency pair",
+		},
+	},
+}
+
+func getOrderbookStream(c *cli.Context) error {
+	if c.NArg() == 0 && c.NumFlags() == 0 {
+		cli.ShowCommandHelp(c, "getorderbookstream")
+		return nil
+	}
+
+	conn, err := setupClient()
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+
+	var exchangeName string
+	var base string
+	var quote string
+	var assetType string
+
+	if c.IsSet("exchange") {
+		exchangeName = c.String("exchange")
+	} else {
+		exchangeName = c.Args().First()
+	}
+
+	if c.IsSet("base") {
```

Comment:
Can put the pair into a single string as per the rest of the commands in gctcli. See this as an example: https://github.com/thrasher-corp/gocryptotrader/blob/5ddad96b518c97774634c395c08a213347b6b7e4/cmd/gctcli/commands.go#L521

The default delimiter is "-" and the above will check to make sure it is valid or not:

https://github.com/thrasher-corp/gocryptotrader/blob/5ddad96b518c97774634c395c08a213347b6b7e4/cmd/gctcli/main.go#L81